packages/scm/src/browser/style/index.css

```diff
@@ -173,48 +173,49 @@
     padding: 0px calc(var(--theia-ui-padding)/2);
 }
 
-.theia-scm .scmItem:hover {
+.scmItem:hover {
     background-color: var(--theia-list-hoverBackground);
     color: var(--theia-list-hoverForeground);
     cursor: pointer;
 }
 
-.theia-scm:focus-within .scmItem:focus {
+*:focus-within .scmItem:focus {
     background: var(--theia-list-focusBackground);
     color: var(--theia-list-focusForeground);
 }
 
-.theia-scm:not(:focus-within) .scmItem:not(:focus) {
+*:not(:focus-within) .scmItem:not(:focus) {
     background: var(--theia-list-inactiveFocusBackground);
 }
 
-.theia-scm:focus-within .scmItem.theia-mod-selected {
+*:focus-within .scmItem.theia-mod-selected {
```

Comment:
It looks bogus,
- top-level classes should follow our coding guidlines and start with `theia-` prefix
- another element is focused within body element and scm item is highlighted as selected?

Reply:
OK, I've fixed that by having both theia-scm and theia-git classes set on the git widgets.